src/i18n/langs/en.js

```diff
@@ -163,7 +163,7 @@ const en = {
     max_display: 'Maximum number of displays:  {num}',
     max_scan: 'Maximum number of scans:  {num}',
     close_left: 'Close Left Tabs',
-    close_right: 'Close Right Tabs',
```

Comment:
I just think your translation isn't detailed enough, for example, capitalization issues, or "Close Left Tabs" in the same level

Reply:
> I just think your translation isn't detailed enough, for example, capitalization issues, or "Close Left Tabs" in the same level
Hi I had just seen this one, I would revert it back to Close Right Tabs. 
My IDE had done it. But mainly it was to correct the lines 29-35: https://github.com/qishibo/AnotherRedisDesktopManager/pull/1103/files#diff-e1881e29c972879b3dbde4cdd7dd38fbd8b5184c1cc39e7e58fd82a7da997badR29